content/administration/upgrade.rst

```diff
@@ -1,264 +1,314 @@
-  subscription**.
+Which version of Odoo should I upgrade to ?
+===========================================
 
-Upgrade services are limited to the technical conversion and adaptation of a database (standard
-modules and data) to make it compatible with the version targeted by the upgrade.
+We always recommend upgrading **to the latest version of Odoo** to benefit from the latest features, improvements, and security fixes. If you are hosted on Odoo Online, you will automatically be enrolled in the rolling release program and will always be on the latest version of Odoo if your database can be upgraded automatically.
 
-Upgrade services not covered by the SLA
----------------------------------------
+.. _upgrade_faq/rolling_release:
 
-The following upgrade-related services are **not** included:
+What is the Rolling Release (applicable to Odoo Online databases) ?
+===================================================================
 
-- the **cleaning** of pre-existing data and configurations while upgrading;
-- the upgrade of **custom modules created in-house or by third parties**, including Odoo partners;
-- lines of **code added to standard modules**, i.e., customizations created outside the Studio app,
-  code entered manually, and :ref:`automated actions using Python code
-  <studio/automated-actions/action>`; and
-- **training** on using the upgraded version's features and workflows.
 
-.. note:: |assistance-contact|
+This feature allows customers to upgrade their database directly from a message prompt sent to the database administrator as soon as the new version is released. Odoo first tests the upgrade to the next version and if no issues are detected, the database administrator receives a message with 2 options:
 
-.. seealso::
-   - :doc:`Upgrade FAQ <upgrade/faq>`
-   - :doc:`Odoo.sh documentation <odoo_sh>`
-   - :doc:`Supported Odoo versions <maintain/supported_versions>`
+#. To 'Upgrade Now', which immediately triggers the upgrade of your live production database. (Discouraged)
+
+#. To take you to your `database manager <https://www.odoo.com/my/databases/>`_ where you can `request an upgraded test database <https://upgrade.odoo.com/#online/>`_ and check the upgraded test database for any discrepancies. (Recommended)
+
+If a database cannot be successfully upgraded automatically, then a ticket is automatically created to allow the upgrade technicians of Odoo to investigate the issue and fix it. Once the issue is fixed, the database is unblocked from the process and is available to be tested automatically again
+
+.. note::
+    In case of a database running a version that will soon be unsupported, the database can be automatically upgraded to the latest supported version of Odoo through the rolling release process.
+
+
+How is my data handled in the Upgrade Platform?
+===============================================
+
+The Odoo Upgrade platform uses the same Privacy Policy as the rest of Odoo.com services.
```

Comment:
Is that the upgrade.odoo.com page? Feels weird to call it a platform, to be honest.
Maybe link the privacy policy.

Reply:
It indeeds refer upgrade.odoo.com, as computer scientist I feel that it is not weird since a platform can refer to any (web) interface where you can submit data. Furthermore on the privacy policy page we can find a similar reference https://www.odoo.com/privacy#:~:text=legacy%20web%2Dbased%20upgrade%20platform%2C :D